src/main.rs

```diff
@@ -1,87 +1,75 @@
-#![allow(dead_code, unused_variables, unused_imports)]
-pub mod accelerator;
+pub mod driver;
```

Comment:
Do you have a recommendation for fixing this? I don't like the idea of putting what is in `driver.rs` now in `drivers/mod.rs` especially now that it has the accelerator functions too. My preferred scenario is rename `drivers` to `implementations` or `features` or something like that.

I don't believe it's too big a deal though.